layers/+tools/org-download/packages.el

```diff
@@ -0,0 +1,22 @@
+;;; packages.el --- org-download layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Laurent Lejeune <olol85@gmail.com>
+;; URL: https://github.com/krakapwa
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+;;; Code:
+
+(defconst org-download-packages
+  '(
+    org
+    org-download
+    )
```

Comment:
Also I think this belongs to `org` layer itself rather than a separate layer. Let's see what others say.